ansible/playbook.retry

```diff
@@ -1 +0,0 @@
-default
```

Comment:
I'm not sure why playbook.retry is even committed. I always thought it was ignored by git.